README.md

```diff
@@ -103,12 +103,14 @@ I also like to add my public key to the `~/.ssh/authorized_keys` file on the rem
 First `ssh` into the onboard computer and `cd path/uploaded/to`.
```

Comment:
I'm wondering if at some point we should redirect to https://github.com/littlebee/basic_bot/tree/main/docs/Installation%20Guides

Also I need another issue/task to flash a fresh Pi4 Bullseye and go through the installation there.   And then pull out the Nvidea Jetson followed by the Corel ?? whatever their SBC is called I have an old one.